packages/wallet/src/redux/protocols/concluding/instigator/reducer.ts

```diff
@@ -138,10 +138,10 @@ function handleLedgerUpdateAction(
     };
   } else {
     return {
-      protocolState: states.instigatorWaitForLedgerUpdate({
```

Comment:
This change seems to make sense, but when I tried reversing it, the new test still passed. Am I missing something?

Reply:
Oops, I was using the wrong action in the scenario. Good catch! Fixed in f92cb9a8bb2bf42b6e0d864ae5b7b58d987c4178